src/app/components/pages/Post.jsx

```diff
@@ -63,7 +65,47 @@ class Post extends React.Component {
         }
         const dis = content.get(post);
```

Comment:
It's kind of gross that this happens in `render`. But I know that was already like that. We should clean it up later.